lib/stradivari/filter/helpers.rb

```diff
@@ -47,6 +47,21 @@ def self.render_title(name, title, opts)
         end
       end
 
+      def self.renderable_field(attribute, value, opts, bound)
+        as_what = opts[:as]
+        clazzez = opts[:class] || ""
```

Comment:
```suggestion
        classes = opts[:class] || ""
```

or

```suggestion
        css_classes = opts[:class] || ""
```

Reply:
Also probably `|| ''` is not needed, because `#{nil}` will render `''` (`nil.to_s`)